tests/contrib/langchain/cassettes/langchain_community/langchain_qdrant_similarity_search_with_score.yaml

```diff
@@ -0,0 +1,67 @@
+interactions:
```

Comment:
Ditto for qdrant cassettes, looks like we can reuse cassettes instead of creating new ones for each test case